main.js

```diff
@@ -106,44 +106,61 @@ function copy_mod_message()
 /* If framerate is invalid, show an error message and disable start and end frame fields. */
 function check_fps(event)
 {
+	const s_time = document.getElementById("startobj");
+	const e_time = document.getElementById("endobj"); 
```

Comment:
The commit is titled "Fixed an issue with the framerate causing inaccurate times" but this isn't fixing that issue, it's just code reorganisation. Same goes for the other typo fix in one of the comments. If you can do the following then I'll be happy to merge this:

 - Move typo fixes to their own commit
 - Move code reorganising/refactoring into its own commit
 - Don't make the link to `index.html` link to `/`